packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -248,23 +273,33 @@ export class AggregatedAttestationPool {
       attestationGroupsArr = Array.from(attestationGroupsByIndex.values());
     }
 
-    const attestations: phase0.Attestation[] = [];
+    const attestations: allForks.Attestation[] = [];
     for (const attestationGroups of attestationGroupsArr) {
       for (const attestationGroup of attestationGroups.values()) {
         attestations.push(...attestationGroup.getAttestations());
       }
     }
     return attestations;
   }
+
+  /**
+   * Electra and after: Block proposer consolidates attestations with the same
+   * attestation data from different committee into a single attestation
+   * https://github.com/ethereum/consensus-specs/blob/aba6345776aa876dad368cab27fbbb23fae20455/specs/_features/eip7549/validator.md?plain=1#L39
+   * TODO Electra: implement this or consider other strategy
```

Comment:
need to do on chain aggregation via `MatchingDataAttestationGroup` before `getAttestationsForBlock` call